### PR TITLE
Address ansible-lint errors

### PR DESCRIPTION
**Pull request by gravesm**
##### SUMMARY

This adds ansible-lint to the linters tox environment, and makes the necessary changes to get the collection to pass.

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by jillr — APPROVED** [review]
lgtm with the one tiny typo updated - thanks!

File: playbooks/roles/scale_virtual_machine/tasks/create.yml [hunk 1]
```diff
@@ -36,41 +37,42 @@
...
+        backend_address_pool: "{{ azure_load_balancer.backend_address_pool }}"
         protocol: Tcp
         frontend_port: 5000
         backend_port: 5000
         probe: ping_server
   register: load_balancer
 
 # Create Bastion Host
-- include_tasks: tasks/create_bastion.yml
+- name: Creat a bastion host
```
[the remark this answers is not in the document]
Comment (gravesm) [reply playbooks/roles/scale_virtual_machine/tasks/create.yml:70]: Those are some eagle eyes!

**Comment by alinabuzachis** [comment]
I guess these kind of warnings: `WARNING  Unable to resolve FQCN for module azure.azcollection.azure_rm_virtualmachinescalesetinstance_info` might be solvable by adding azure.azcollection as a dependency in the galaxy file.

**Comment by alinabuzachis** [comment]
> I guess these kind of warnings: `WARNING Unable to resolve FQCN for module azure.azcollection.azure_rm_virtualmachinescalesetinstance_info` might be solvable by adding azure.azcollection as a dependency in the galaxy file.

Ah, there is already.

**Pull request merged**